test/parallel/test-eventtarget-whatwg-once.js

```diff
@@ -0,0 +1,92 @@
+// Flags: --expose-internals
```

Comment:
Put a link to the wpt source with a comment on why this was manually ported rather than using `git node wpt test` :]